java/ql/src/Security/CWE/CWE-798/HardcodedAzureCredentials.java

```diff
@@ -0,0 +1,52 @@
+public class HardcodedAzureCredentials {
```

Comment:
File no longer used

Reply:
Removed the sample Java file together with the <code>.qhelp</code> help file